### PR TITLE
This is test commit

### PR DESCRIPTION
**Pull request by shrijan101**
## Description 

this is test commit to test github pull request

### CODE REVIEW

File: src/components/GalleryList.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ export default function GalleryList() {
               <td>{object.title}</td>
               <td>{object.totalImages}</td>
               <td>
+                <td>hello world</td>
```
Comment (bibekmaharjan) [inline src/components/GalleryList.js:35]: This change is not required please remove

File: src/components/GalleryList.js
Comment (bibekmaharjan) [inline src/components/GalleryList.js:1]: The use of useState should also include setState for future when the mock data is replaced with the data fetched with API or if the data change is needed.

**Comment by bibekmaharjan** [comment]
This change is not required so closing the PR and deleting the branch

**Comment by shrijan101** [comment]
reopening for test

**Pull request closed without merging**